### ISSUE TITLE
Gadla Ewostatewos

### ISSUE THREAD

**Issue by nafisa-valieva** [SChAth/ScAthiop#2707]
Hello! While working on a book review about Ewostatewos, I came across the following confusion in Bm:
based on Lusini´s article, Bm correctly distinguishes 3 redactions, but in the absence of definition, there are some problems:

(A) LIT4319GadlaEA which is said to be contained in Aeth. 46 and BL Oriental 702 (in MS BL Oriental 702 description the LIT ID is chosen wrongly as LIT4321)

Wrongly encoded as edited by Turaiev, B. 1905. 

(B) LIT4320GadlaEB which is said to be contained in BL Oriental 704 and BL Oriental 705

encoded as having no edition, but in reality the one edited by Turaev

(C) LIT4321GadlaEC which is said to be contained in BL Oriental 703 

I want to correct these errors (and related errors), add abstracts and, if possible, add incipits and explicits. 
Besides, I have a general question: there are two general IDs, LIT3200DossierEw and LIT1438GadlaE.
Should these general IDs contain edition bibl? I would say no, but tell me if I am wrong. If it is common to have an edition bibl for general records too, what would be the technical way to explain what one should expect to find in the edition(s)?
Thank you very much!

**Comment by CarstenHoffmannMarburg** [comment]
I would opt to list the literature according to their relevance, e.g. secondary literature, that deals with both recensions in the general/generic record, which is for both recensions. Editions and secondary literature, that deals specifically with one recension only in the record for this recension. Editions, which present the text of different recensions in parallel should go to each of the recensions records.

**Comment by eu-genia** [comment]
to me it is clear: 
bibliography dealing with the dossier, that is the collection containing gadl+miracles+hymn should go under `LIT3200DossierEw `
bibliography dealing with the Gadl in all versions (or non-specified version) should go under `LIT1438GadlaE `
neither of these records should have text though
individual recensions should have bibliography and text for the specific recensions

**Comment by nafisa-valieva** [comment]
> to me it is clear: bibliography dealing with the dossier, that is the collection containing gadl+miracles+hymn should go under `LIT3200DossierEw ` bibliography dealing with the Gadl in all versions (or non-specified version) should go under `LIT1438GadlaE ` neither of these records should have text though individual recensions should have bibliography and text for the specific recensions

good! then I will correct general records too, if no one minds.

**Comment by MassimoVilla** [comment]
I agree with @eu-genia's distribution of the information among the various records.
However, as far as I remember Turaev's edition is eclectic, he uses MSS which were later classified as belonging to different recensions. I can't check now.

**Comment by nafisa-valieva** [comment]
> I agree with @eu-genia's distribution of the information among the various records. However, as far as I remember Turaev's edition is eclectic, he uses MSS which were later classified as belonging to different recensions. I can't check now.

thank you! I did check, he uses mainly > BL Oriental 704 and BL Oriental 705. I will try to find a way to explain what he did.
